CHANGES.md

```diff
@@ -87,6 +87,10 @@ To be released.
 
 ### CLI tools
 
+ -  `planet mpt diff` command became to support also comparing
+    two state root hashes from two different key-value store.
+    [[#1129]]
```

Comment:
The new required argument in the middle makes the existing commands no more work.  So it should be noted in the changelog:

```suggestion
 -  `planet mpt diff` command became to take 4 arguments (which was 3) so that it can
    compare state root hashes from two different <abbr title="key–value">KV</abbr>
    stores.  The existing commands like `planet mpt diff STORE A B` do not work anymore,
    and these should be instead like `planet mpt diff STORE A STORE B`.
    [[#1129]]
```

Reply:
I applied it in dbf5c72